eos-updater-flatpak-installer/eos-updater-flatpak-installer.service.in

```diff
@@ -1,12 +1,12 @@
 [Unit]
 Description=Endless OS Post-Boot Flatpak Installer
-Wants=local-fs.target
-After=local-fs.target
+Wants=network.target
+After=network.target
 ConditionKernelCommandLine=!eos-updater-disable
+DefaultDependencies=no
```

Comment:
I don't think you can specify `After=network.target` for an update service. `systemd-update-done.service` has `Before=sysinit.target`. That conflicts with `NetworkManager.service`, which is a normal service that runs `After=sysinit.target`. That creates the ordering cycle that systemd breaks. And if it removes `systemd-update-done.service`, then there's not guarantee this service will run before the `.updated` files are created.

Reply:
Yeah, you're right - there was a conflict. Thankfully, there is `ostree-remount.service` which gets us what we need in terms of filesystem setup.